prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -1167,60 +1110,6 @@ periodics: # runs on schedule
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
-    - name: serverless-function-metrics-generator
```

Comment:
Please keep it. 
@kyma-project/otters will fix it
https://github.com/kyma-project/test-infra/issues/7684

Reply:
Restored

Reply:
Fixed: https://github.com/kyma-project/test-infra/pull/7685